app/controllers/admin/settings/authentications_controller.rb

```diff
@@ -1,15 +1,10 @@
 module Admin
   module Settings
-    class AuthenticationsController < Admin::ApplicationController
-      def create
-        result = ::Authentication::SettingsUpsert.call(settings_params)
+    class AuthenticationsController < Admin::Settings::BaseController
```

Comment:
This and further controllers got simplified by using the new `Admin::Settings::BaseController`. Some now are essentially empty because they can rely completely on the default implementations.